source/jormungandr/tests/keolis_tests.py

```diff
@@ -42,12 +42,14 @@ def check(profile, type):
         journey_direct = response.journeys.add()
 
         journey_direct.type = "best"
+        print 'journey_direct', journey_direct.type
+
         #we set the destination else the filter will not be applicated
         req = {'debug': False, 'destination': 'foo'}
-        scenario._qualification(req, response, travelers_profile[profile])
+        scenario._qualification(req, response, default_traveler_profiles[profile])
         eq_(type, journey_direct.type)
 
-    for profile in travelers_profile.keys():
+    for profile in default_traveler_profiles.keys():
```

Comment:
write it this way:
  for profile in default_traveler_profiles:
this is more portable...